packages/jest-environment-enzyme/package.json

```diff
@@ -9,8 +9,9 @@
   "scripts": {
     "lint": "eslint ./src",
     "clean": "rm -rf ./lib",
-    "build:js": "babel src --out-dir lib --ignore tests --copy-files",
-    "build": "npm run clean && npm run build:js",
+    "build:js": "babel --root-mode upward src --out-dir lib --ignore '**/__tests__/*.js'",
+    "copy-dts": "cpy src/index.d.ts lib",
```

Comment:
https://github.com/babel/babel/issues/6226
> Bug: If in your build script, you have --copy-files passed, files that are ignored in .babelrc files will be copied over, instead of being ignored.